ompi/communicator/communicator.h

```diff
@@ -233,7 +234,7 @@ typedef struct ompi_communicator_t ompi_communicator_t;
  * This is done so we don't end up needing a structure that is
  * incredibly larger than necessary because of the bitness.
  */
-#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 192)
+#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 194)
```

Comment:
Well, when *I* read all of that, it seems like you shouldn't EVER change the PREDEFINED_COMMUNICATOR_PAD (since that was set long ago), and that the padding will auto-shrink as you add fields to ompi_communicator_t), though perhaps we could reset it at a MAJOR.0.0 release (since we require users to recompile/relink at that time).

So, I still don't understand how you can change the PREDEFINED_COMMUNICATOR_PAD without breaking backwards compatibility.